src/Avalonia.Base/Utilities/DispatcherTimerHelper.cs

```diff
@@ -0,0 +1,18 @@
+using Avalonia.Threading;
+
+namespace Avalonia.Utilities;
+
+public class DispatcherTimerHelper
+{
+    
+}
+
+public static class DispatcherTimerUtils
+{
+    public static void ForceFire(this DispatcherTimer timer)
```

Comment:
Empty class.

Also, just for case, do we keep it public?
I think alternatively it could be internal here with public copy in Headless assembly. Either way, I am fine with it.

Reply:
It was supposed to go to Avalonia.Base.UnitTests, will fix